ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.9__4.5.yaml

```diff
@@ -0,0 +1,84 @@
+base_images:
+  base:
+    name: "4.5"
+    namespace: ocp
+    tag: base
+  cli:
+    name: "4.5"
+    namespace: ocp
+    tag: cli
+build_root:
+  project_image:
+    dockerfile_path: openshift/ci-operator/build-image/Dockerfile
+images:
+- context_dir: openshift/ci-operator/source-image
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+  to: serverless-operator-src
+- context_dir: serving/ingress
+  from: base
+  to: knative-openshift-ingress
+- context_dir: knative-operator
+  from: base
+  to: knative-operator
+- context_dir: olm-catalog/serverless-operator
+  from: base
+  to: serverless-bundle
+promotion:
+  name: openshift-serverless-v1.9.0
+  namespace: openshift
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.5"
+  namespace: ocp
+tests:
+- as: e2e-aws-ocp-45
+  steps:
+    cluster_profile: aws
+    test:
+    - as: serverless-e2e
+      commands: make test-e2e
```

Comment:
This will only run the "operator" tests from serverless-operator repo. Not all E2E. ...looking at the setup for "master" branch should give you more info about the make targets etc.

Reply:
I did _borrow_ this from 1.8:

https://github.com/matzew/release/blob/add_serverless_1.9/ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.8__4.5.yaml#L48

Reply:
Yeah. It has changed since 1.8 branch. The changes you've just pushed should fix that.